ejb-txn-remote-call/README.adoc

```diff
@@ -0,0 +1,942 @@
+include::../shared-doc/attributes.adoc[]
+
+= ejb-txn-remote-call: Demonstrates remote EJB calls and transaction propagation
+:author: Ondra Chaloupka
+:level: Intermediate
+:technologies: EJB, JTA, Clustering
+
+[abstract]
+The `ejb-txn-remote-call` quickstart demonstrates the remote EJB calls over two application servers of {productName}.
```

Comment:
"demonstrates the remote EJB calls"

The article is not necessary; delete "the"

"remote EJB calls over two application servers "

Is this quickstart illustrating a high-availability environment? If not, the preposition should change: "over" > "between".

Reply:
Yes, the quickstart should illustrate kind of the high-availability environment where the `server1` calls one of two other servers `server2` and/or `server3`. If on of the other servers fail then there is still one active available to process requests. Is it fine to leave here the "over" then?